.mergify.yml

```diff
@@ -3,33 +3,30 @@
 queue_rules:
   - name: default
     conditions:
-      - '#approved-reviews-by>=2'
```

Comment:
Do we want to drop this?

Reply:
We don't have it in the other projects, I only included it in the original version so the conditions wouldn't be empty. The important place is to have it in the merge_requests_rules below.